httpapi/metrics.go

```diff
@@ -7,20 +7,10 @@ import (
 func init() {
 	prometheus.MustRegister(metricFunctionRegistered)
 	prometheus.MustRegister(metricFunctionDeleted)
```

Comment:
@mthenw Would it be better to have a single `metricFunctions` that is just a gauge of registered functions for a Space?

Reply:
Makes sense!